### PR TITLE
Fix circle

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,6 +14,6 @@ jobs:
     steps:
       - run: apk add --no-cache openldap-dev cyrus-sasl-dev git build-base
       - checkout
-      - run: pip install -r requirements.txt
+      - run: pip install -r /useradm/requirements.txt
```
Comment (butlerx) [inline .circleci/config.yml]: it would probably be better to change the working dir to `~/useradm/useradm`

**Comment by gruunday** [comment]
Yeah I am stupid

**Comment by butlerx** [comment]
itd be better to change https://github.com/redbrick/useradm/pull/28/files#diff-1d37e48f9ceff6d8030570cd36286a61L5 to `~/useradm/useradm`

**Comment by gruunday** [comment]
Not for a test folder at the same level as useradm. Which I am currently writing

**Review by butlerx — APPROVED** [review]
fair enough then

**Pull request merged**